actors/builtin/paych/paych_test.go

```diff
@@ -0,0 +1,179 @@
+package paych_test
+
+import (
+	"context"
+	"testing"
+
+	addr "github.com/filecoin-project/go-address"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	. "github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	"github.com/filecoin-project/specs-actors/actors/crypto"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/filecoin-project/specs-actors/support/mock"
+	tutil "github.com/filecoin-project/specs-actors/support/testing"
+)
+
+func TestPaymentChannelActor_Constructor(t *testing.T) {
```

Comment:
commentary: this would make a good table driven test ([example](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/account/account_test.go#L30))

Reply:
yes... this form hasn't quite become habitual, but I have been trying to use it more.